boot/zephyr/uuid/CMakeLists.txt

```diff
@@ -0,0 +1,71 @@
+#
+# Copyright (c) 2025 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+if(CONFIG_NCS_MCUBOOT_UUID_SINGLE_VID)
+  if(CONFIG_MCUBOOT_UUID_VID OR CONFIG_MCUBOOT_UUID_CID)
+    zephyr_library_sources(
+      uuid.c
+      )
+  endif()
+
+  # Generate VID value and raw value definition
+  if(CONFIG_MCUBOOT_UUID_VID OR CONFIG_MCUBOOT_UUID_CID)
+    if("${CONFIG_NCS_MCUBOOT_UUID_VID_VALUE}" STREQUAL "" AND CONFIG_MCUBOOT_UUID_VID)
+        message(FATAL_ERROR "VID value not set")
```

Comment:
2 space indent for cmake, also do not throw fatal error here (fine if the code itself does not compile) so that users can fix the issue without the whole cmake build being broken